chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDefinition.java

```diff
@@ -100,6 +104,17 @@ public Builder withQueryProvider(Iterable queryProvider) {
         }
 
         /**
+         * Optional: Sets load balancer (subtypes of {@link QueryPoolLoadBalancer}).
+         * Default is {@link RoundRobinLoadBalancer}
+         *
```

Comment:
If you have multiple clients every of them have to check on null that field if any of them forgets it gets NPE. So, more code and possibility of NPE